src/info.json

```diff
@@ -1,11 +1,26 @@
 {
     "name": "subspace_storage",
-    "version": "1.99.4",
+    "variants": [
+        {
+            "version": "1.99.6",
```

Comment:
Why do you label different patch versions for the different factorio versions? The code inside is the same, except for the compatibility layer and they are released at the same time. Is it a portal limitation related to not uploading multiples of the same version? In that case, 1.99.6.018/1.99.6.111 etc might make more sense (if thats allowed).

Reply:
I'm pretty sure you're not allowed to upload multiple zips of the same version and the game also ignores a 4th number for the mod version so I suspect the mod portal would too.  So for now it's patch increments.  Might decide to go for 0.17 -> 2.0.x, 1.0 -> 2.1.x, 1.1 -> 2.2.x and so on in the future when it's no longer alpha.

Reply:
On the other hand.  Using patch versions like this makes the version history chronological, higher version is always newer.

Reply:
I think "0.17 -> 2.0.x, 1.0 -> 2.1.x, 1.1 -> 2.2.x" is completely nonsensical, because as you say that would make the version history non chronological.

If we aren't doing a 4th number, how about 2.x.y017, 2.x.y100, 2.x.y110? Keeps it chronological, and some indication as to what version each build is for. Not particularly pretty though.

Reply:
But do you need to know which patch version is for which Factorio version by looking at it?  If you look at the downloads page for a mod it already displays that ([see example](https://mods.factorio.com/mod/subspace_storage/downloads)).  And the in game mod menu only lets you install the latest compatible version.

Reply:
When I go to update the mod and see that I am on 1.99.6 and the latest is 1.99.12 I expect there to have been a lot of changes in between, not 2. Also clears up the holes in the changelog, which can be confusing. Its all just semantics though, if you don't feel like changing it thats fine.

Reply:
What about the 999 releases skipped between 2.1.1101 and 2.1.2101?  It's difficult to see what the numbers mean looking at them at a glance.